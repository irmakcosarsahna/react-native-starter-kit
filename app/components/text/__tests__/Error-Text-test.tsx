/**
 * @format
 */

import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import { Text } from '../index';

it('renders Error Text', () => {
  const component = renderer.create(<Text text={'error test'} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
