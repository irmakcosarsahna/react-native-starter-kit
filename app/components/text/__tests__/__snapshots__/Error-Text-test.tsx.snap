// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Error Text 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "rgb(215, 38, 61)",
        "fontFamily": "Arial",
        "fontSize": 15,
      },
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  error test
</Text>
`;
